ygen/gogen.go

```diff
@@ -618,6 +618,35 @@ func (t *{{ .Receiver }}) GetOrCreate{{ .ListName }}(
 	}
 	return v
 }
+`
+
+	// goDeleteListTemplate defines a template for a function that, for a
+	// particular list key, deletes an existing map value.
+	goDeleteListTemplate = `
+// Delete{{ .ListName }} deletes the value with the specified keys from
+// the receiver {{ .Receiver }}.
```

Comment:
Is it worth clarifying here that the behaviour is the same as the `delete` builtin, since there is no check for the key's existence in the list?

Reply:
Done.